core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -562,41 +643,72 @@ func (c *coordinator) ReportWillBeTransmitted(ctx context.Context, report ocr2vr
 	c.transmittedMu.Lock()
 	defer c.transmittedMu.Unlock()
 
-	blocksRequested := make(map[block]struct{})
-	blockNumbersRequested := []uint64{}
-	callbacksRequested := []callback{}
+	now := time.Now().Unix()
+
+	// Evict expired items from the cache.
+	c.toBeTransmittedBlocks.EvictExpiredItems(now)
+	c.toBeTransmittedCallbacks.EvictExpiredItems(now)
```

Comment:
Do we need to have this if the interval cleaner is already calling `EvictExpiredItems` periodically?

Reply:
_Technically_ we don't, but I would want to make the expiration interval much faster in the cleaner if we get rid of this logic, as if it doesn't clean up items frequently enough then they will be needlessly be dropped by report blocks (even though they should be expired).